digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -21,7 +21,7 @@ sealed trait ProjectState {
 object ProjectState {
 
   val values = Set(Closed, Incomplete, Sent2TR, TRProcessing, Saved2TR,
-    Deleted, ErrorInViite, SendingToTR, Unknown)
+    Deleted, ErrorInViite, SendingToTR, InUpdateQueue, UpdatingToRoadNetwork, Unknown)
```

Comment:
Where is the queue? Could it be pendingToUpdateRoadNetwork?
But the queue is coming one day, so it could be like this.

Reply:
You are sure the real queue WILL be there one day? :D (I'm always a bit sceptical.. All the "We'll do that then one day" stuff tends to go down to the bottom of the ToDo-list, and will usually never be revisited again, if it is not mandatory for the functionality... :D :'(

To the question:
The "queue" is currently just in the DB. It is, the projects the users have accepted - having just this ProjectStatus in question - and the DB is asked to give a single project at a time.
But very good, the proposed name would currently be better, and wouldl still be true (though not that explicit then than that in the PR now), if we sometimes end up putting the real queue in the AWS/SQS).
(Though, I would add another "To" there: pendingToUpdate_To_RoadNetwork :D )

More opinions about this ProjectStatus name, anyone?